fastquant/disclosures.py

```diff
@@ -73,10 +75,13 @@ def __init__(
             start date with format %m-%d-%Y
         end_date : str
             end date with format %m-%d-%Y
+        start_page : int
+            first disclosure page to start with
         """
         self.symbol = symbol.upper()
         self.start_date = start_date
         self.end_date = TODAY if end_date is None else end_date
+        self.start_page = start_page
```

Comment:
This always starts at page 1, isn't it? Is there a case otherwise?

Reply:
Actually good point. Will just set it to 1